doc/node.1

```diff
@@ -142,68 +113,65 @@ underscores (\fB_\fR).
 
 For example, \fB\-\-stack\-trace\-limit\fR is equivalent to
 \fB\-\-stack\_trace\_limit\fR
-
 .TP
 .BR \-\-v8\-pool\-size =\fInum\fR
 Set v8's thread pool size which will be used to allocate background jobs.
 If set to 0 then v8 will choose an appropriate size of the thread pool based
 on the number of online processors. If the value provided is larger than v8's
 max then the largest value will be chosen.
-
 .TP
 .BR \-\-tls\-cipher\-list =\fIlist\fR
 Specify an alternative default TLS cipher list. (Requires Node.js to be built
 with crypto support. (Default))
-
 .TP
 .BR \-\-enable\-fips
 Enable FIPS-compliant crypto at startup. (Requires Node.js to be built with
 \fB./configure \-\-openssl\-fips\fR)
-
 .TP
 .BR \-\-force\-fips
 Force FIPS-compliant crypto on startup. (Cannot be disabled from script code.)
 (Same requirements as \fB\-\-enable\-fips\fR)
-
 .TP
 .BR \-\-icu\-data\-dir =\fIfile\fR
 Specify ICU data load path. (overrides \fBNODE_ICU_DATA\fR)
-
-.SH ENVIRONMENT VARIABLES
-
+.SH ENVIRONMENT
 .TP
 .BR NODE_DEBUG =\fImodule\fR[,\fI...\fR]
 \',\'\-separated list of core modules that should print debug information.
-
 .TP
 .BR NODE_PATH =\fIpath\fR[:\fI...\fR]
 \':\'\-separated list of directories prefixed to the module search path.
-
 .TP
 .BR NODE_DISABLE_COLORS=1
 When set to 1 colors will not be used in the REPL.
-
 .TP
 .BR NODE_ICU_DATA =\fIfile\fR
 Data path for ICU (Intl object) data. Will extend linked-in data when compiled
 with small\-icu support.
-
 .TP
 .BR NODE_REPL_HISTORY =\fIfile\fR
 Path to the file used to store the persistent REPL history. The default path
 is ~/.node_repl_history, which is overridden by this variable. Setting the
 value to an empty string ("" or " ") disables persistent REPL history.
-
 .TP
 .BR NODE_TTY_UNSAFE_ASYNC=1
 When set to 1, writes to stdout and stderr will be non-blocking and asynchronous
 when outputting to a TTY on platforms which support async stdio.
 Setting this will void any guarantee that stdio will not be interleaved or
 dropped at program exit. \fBAvoid use.\fR
-
-
+.SH BUGS
+Bugs are tracked in GitHub Issues:
+.ur https://github.com/nodejs/node/issues
+.SH AUTHORS
+Written and maintained by 1000+ contributors:
+.ur https://github.com/nodejs/node/blob/master/AUTHORS
+.SH COPYRIGHT
+Copyright Node.js contributors. All rights reserved. Node.js is available under the MIT license.
```

Comment:
I don't think this correct -- all rights are not reserved, it is licensed under MIT.